src/main/java/com/gtnh/findit/FindItConfig.java

```diff
@@ -82,6 +83,14 @@ public static void setup(final File file) {
                             "FFFF8726",
                             "Item highlighting color as a hexadecimal color code. For example 0xFFFF8726").getString(),
                     16);
+
+            ITEM_HIGHLIGHTING_EMPTY_ITEMSTACKS = config.get(
+                    Configuration.CATEGORY_GENERAL,
+                    "ItemHighlightingEmptyItemStacks",
+                    "false",
```

Comment:
I think it should be true by default. We have ghost circuit slot in GT machines, which is 0 sized stack.